src/state/persisted/schema.ts

```diff
@@ -7,9 +7,9 @@ const accountSchema = z.object({
   did: z.string(),
   refreshJwt: z.string().optional(),
   accessJwt: z.string().optional(),
-  handle: z.string(),
-  displayName: z.string(),
-  aviUrl: z.string(),
+  handle: z.string().optional(),
+  displayName: z.string().optional(),
+  aviUrl: z.string().optional(),
```

Comment:
If our plan is to fetch this data to render screens that need it, should we just remove these entirely and only store the bare minimum in storage?

Reply:
that might be better yeah